src/Native/Runtime/EHHelpers.cpp

```diff
@@ -133,8 +133,12 @@ COOP_PINVOKE_HELPER(void, RhpCopyContextFromExInfo,
     pContext->Rip = pPalContext->IP;
     pContext->Rsp = pPalContext->Rsp;
     pContext->Rbp = pPalContext->Rbp;
+#ifndef UNIX_AMD64_ABI
```

Comment:
I'm not sure I'm a fan of #ifdefing under the _AMD64_ block -- I think I'd rather see us treat amd64 unix as aseparate 'target cpu' and have the whole block separated per platform, i.e. one for _AMD64_ && UNIX_AMD64_ABI  and one for _AMD64_ && !UNIX_AMD64_ABI